src/Goerli-DssSpell.sol

```diff
@@ -16,18 +16,73 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-// pragma experimental ABIEncoderV2;
+pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-// Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-pragma experimental ABIEncoderV2;
+import {Initializable} from "dss-exec-lib/DssExecLib.sol";
```

Comment:
Let's avoid interface imports in spells in favor of explicit interfaces.

Reply:
Done